### DISCUSSION TITLE
Авто-запуск и отслеживание постов.

### DISCUSSION THREAD

**Discussion by ObsessionD** [MrZillaGold/VK2Discord#551] [Q&A] [answered]
Доброго времени суток. Для начала.. спасибо за ваш труд, это очень классная вещь.

Хотелось бы поинтересоваться, можно ли сделать так:
1. Как-то создать программу чтобы она зпускалась при запуске компьютера (не обязательно) и сворачивалась в иконку на панели задач? Каждый раз прописывать одно и тоже не очень удобно.. точнее совсем неудобно.)
2. Можно ли как-то или через что-то сделать оффлайн режим?
3. Можно ли сделать так, чтобы публиковались посты с учетом того - какой пост был последний? На данный момент получается так, что он проверяет только публиковался или нет последний пост, а если за это время было опубликовано больше постов - он их пропускает.

**Comment by egorprnn** [comment, marked as answer]
1. На Windows можно использовать планировщик задач, как им пользоваться можно найти в Google. Нужно сделать bat файл с командой запуска и добавить задачу для него.
2. Что подразумевается под оффлайн режимом? Работа скрипта если компьютер выключен невозможна, ставьте его на хостинг.
3. Скрипт предназначен для отслеживания постов в реальном времени, можно реализовать функционал публикации пропущенных постов, но это будет не быстро, свободного времени сейчас у меня особо нет.
